src/index.html

```diff
@@ -1,13 +1,518 @@
+                    <p class="hours__weekend-text">Thanksgiving Day, Dec 25, Jan 1, and the first Monday in May.</p>
+                  </div>
+
+                </div>
+              </div>
+            </div>
+          </div>
+        </section>
+
+        <section class="section shows" id="shows">
+          <div class="container">
+            <div class="section__title shows__title">
+              <h2 class="section__title-main section__title-main--center">
+                Now On View
+              </h2>
+              <p class="section__title-sub shows__title-sub">
+                Explore exhibition gallery guides, videos, articles, and more.
+              </p>
+            </div>
+
+            <div class="show shows__item">
+              <div class="show__gallery">
+                <div class="grid">
+                  <div
+                    class="grid__item
+                    grid__item--tablet-1-4
+                    grid__item--desktop-1-8"
+                    >
+                    <div class="show__image-container">
+                      <img
+                        src="images/gallery/gallery-1-2.jpg"
+                        alt="Show Image"
+                        class="show__image"
+                      >
+                    </div>
+                  </div>
+
+                  <div
+                    class="grid__item
+                    grid__item--tablet-5-6
+                    grid__item--desktop-9-12"
+                  >
+                    <div class="show__image-container">
+                      <img
+                        src="images/gallery/gallery-1-1.jpg"
+                        alt="Show Image"
+                        class="show__image"
+                      >
+                    </div>
+                  </div>
+                </div>
+              </div>
+              <div class="show__info">
+                <div class="show__info-name">A New Look at Old Masters</div>
+                <div class="show__info-date">December 12, 2020 – Through Spring 2023</div>
+              </div>
+            </div>
+
+            <div class="show shows__item">
+              <div class="show__gallery">
+                <div class="grid grid--3-columns shows__grid">
+                  <div class="
+                    shows__grid-item
+                    grid__item
+                    grid__item--tablet-1-3
+                    grid__item--desktop-1-4"
+                  >
+                    <div class="show__image-container">
+                      <img
+                        src="images/gallery/gallery-2-1.jpg"
+                        alt="Show Image"
+                        class="show__image"
+                      >
+                    </div>
+                  </div>
+                  <div class="
+                    shows__grid-item
+                    grid__item
+                    grid__item--tablet-4-6
+                    grid__item--desktop-5-8">
```

Comment:
Need to check and fix it everywhere
```suggestion
                  <div
                    class="
                      shows__grid-item
                      grid__item
                      grid__item--tablet-4-6
                      grid__item--desktop-5-8"
                  >
```